fireflower/targets.py

```diff
@@ -28,19 +28,33 @@ class FireflowerS3Target(S3Target):
 
     fs = None
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.local_s3_path = os.getenv('LOCAL_S3_PATH', None)
```

Comment:
Stylistically, would extract `os.getenv('LOCAL_S3_PATH', None)` into class constant, and
`self.local_s3_path = kwargs.get('LOCAL_S3_PATH', LOCAL_S3_PATH)`